db/migrate/20220707153325_create_reservations.rb

```diff
@@ -0,0 +1,11 @@
+class CreateReservations < ActiveRecord::Migration[7.0]
+  def change
+    create_table :reservations do |t|
+      t.date :start_date
+      t.date :end_date
+      t.integer :cost
```

Comment:
- [ ] I think also for reservations we just need the `city` and `date`, what about `cost`, why `end_date` and `start_date`??
- [ ] As Jerry said, there are no associations between the database tables.